src/main/java/com/pusher/client/PusherOptions.java

```diff
@@ -43,24 +43,39 @@
     private int maxReconnectGapInSeconds = MAX_RECONNECT_GAP_IN_SECONDS;
 
     /**
-     * Gets whether an encrypted (SSL) connection should be used when connecting
-     * to Pusher.
-     *
-     * @return true if an encrypted connection should be used; otherwise false.
+     * @deprecated
+     * Please use isForceTLS
      */
+    @Deprecated
     public boolean isEncrypted() {
-        return encrypted;
+        return forceTLS;
+    }
+
+    /**
+     * @deprecated
+     * Please use setForceTLS
+     */
+    @Deprecated
+    public PusherOptions setEncrypted(final boolean encrypted) {
+        this.forceTLS = encrypted;
+        return this;
     }
 
     /**
-     * Sets whether an encrypted (SSL) connection should be used when connecting to
-     * Pusher.
      *
-     * @param encrypted Whether to use an SSL connection
+     * @return whether the connection to Pusher should use TLS
+     */
+    public boolean isForceTLS() {
+        return forceTLS;
+    }
+
+    /**
+     * Sets whether the connection to Pusher should be use TLS.
+     * @param forceTLS whether the connection should use TLS, by default this is true
      * @return this, for chaining
      */
-    public PusherOptions setEncrypted(final boolean encrypted) {
-        this.encrypted = encrypted;
+    public PusherOptions setForceTLS(boolean forceTLS) {
```

Comment:
```suggestion
    public PusherOptions setForceTLS(final boolean forceTLS) {
```

Just for consistency :shrug: